features/collection/thresholding/thresholding-shell/src/main/java/org/opennms/netmgt/threshd/shell/Clear.java

```diff
@@ -29,14 +29,33 @@
 package org.opennms.netmgt.threshd.shell;
 
 import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Option;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
 import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.opennms.netmgt.threshd.api.ThresholdStateMonitor;
 
 @Command(scope = "opennms-threshold-states", name = "clear", description = "Clears a specific threshold state")
 @Service
 public class Clear extends AbstractKeyOrIndexCommand {
```

Comment:
Can we update this command so that both the persisted and in-memory state are cleared by default?